docs/getting-started/index.rst

```diff
@@ -55,66 +55,9 @@ Installation
         :text: Learn more
         :classes: btn-secondary stretched-link
 
-.. _quickstart-setup:
+.. ifconfig:: not 'dev' in version
```

Comment:
Does this mean the the Quickstart is not included at all for dev versions? I guess this mostly makes sense, alternatively we could just work with a place-holder like:

```
https://gammapy.org/download/install/gammapy-x.x.x-environment.yml
```

Reply:
I guess the downside of hiding it for dev version is that we don't see the rendered output even when changes are being made? But I have no strong opinion there...

Reply:
Yes Indeed. With the `ifconfig` scheme you can only test the result on a tagged version, so hiding the section completely is a bit annoying.

Reply:
+1 for the `X.Y.Z` I think this makes it obvious, that there is no env file for dev versions...